docs/tutorials/Soil/richards_equation.jl

```diff
@@ -147,16 +150,35 @@ Y.soil.ϑ_l .= FT(0.494);
 # Next, we turn to timestepping. We choose the initial and final times, as well as a timestep.
 t0 = FT(0)
 tf = FT(60 * 60 * 24 * 36)
-dt = FT(100);
+dt = FT(1e3);
 
 # Now, we choose the timestepping algorithm we want to use.
-# We'll use the 4th order Runge-Kutta method, RK4.
-stepper = ClimaLSM.RK4();
-ode_algo = CTS.ExplicitAlgorithm(stepper);
+# We'll use the ARS111 algorithm with a maximum of 1 Newton iteration
+# per timestep.
```

Comment:
"We'll use the ARS111 algorithm with 1 Newton iteration per timestep; you can also specify a convergence criterion and a maximum number of Newton iterations."